### DISCUSSION TITLE
Using CATALYST without sample barcodes

### DISCUSSION THREAD

**Discussion by suginoka** [github/catalyst#267] [Q&A] [answered]
Hi CATALYST team,

I have some CyTOF sample data I’m trying to process, but I’ve run into an issue when trying to follow the CATALYST tutorial using my samples. My samples were run without barcodes so I have individual fcs files for each sample, which I read into R using list.files() and prepData() commands. I was able to get through the normalization step, but the compensation step uses the output from the debarcoding step, so I am unable to run my sce through the computeSpillmat() command since I do not have barcodes to run the previous debarcoding protocol. 
 
Do you know of a workaround for this? I’ve looked through the example dataset and tutorial outputs and the only thing I can think to do to match my files to the tutorial file format is to create dummy channels in each of my sample files and then create a sample_key that can be run through the debarcoding steps. I’m not great with R, so I have no idea how to do this and would like to avoid directly editing my fcs files if possible. I also already know which events belong to which samples, so there could be some way to utilize this information within the program or data structure I’m just not sure how. 

Thank you!
Kameron

**Comment by keithamus** [comment, marked as answer]
Hey @suginoka! I think the repository you're after is https://github.com/HelenaLC/CATALYST. This is the discussions board for a front-end UI framework that has a similar name.

**Reply by suginoka** [reply]
Hi Keith,

My mistake! Thank you for letting me know!

Best,
Kameron
